packages/contracts-bedrock/tasks/update-registry.ts

```diff
@@ -0,0 +1,62 @@
+import fs from 'fs'
+import path from 'path'
+
+import { task } from 'hardhat/config'
+
```

Comment:
Can you add a comment here describing the purpose and usage of this task? 

Also re: moving to foundry, [this](https://github.com/wighawag/forge-deploy) looks promising to move to foundry in a backwards compat way, would love it if you took a look at it and shared thoughts